ctru-sys/Cargo.toml

```diff
@@ -8,3 +8,11 @@ edition = "2021"
 
 [dependencies]
 libc = { version = "0.2.121", default-features = false }
+doxygen-rs = { git = "https://github.com/Techie-Pi/doxygen-rs.git", version = "0.2.2", rev = "573f483ad63ab4662650961998d3ed093a703983", optional = true }
```

Comment:
It might be preferable to split out `docstring-to-rustdoc` as a separate crate, just to make it very clear what is a "dev dependency" (tooling) vs a real dependency. This is the model used in e.g. [cargo-xtask](https://github.com/matklad/cargo-xtask), and I think I have seen in some other projects as well.

Reply:
I think this is what you meant(?): 95756f7b73cd82994a3257acf31a179b1e1af00b